mino/mod.go

```diff
@@ -59,7 +68,7 @@ type RPC interface {
 type Handler interface {
 	// Process handles a single request by producing the response according to
 	// the request message.
-	Process(req proto.Message) (resp proto.Message, err error)
+	Process(req Request) (resp proto.Message, err error)
```

Comment:
following that, shouldn't we also update Receiver.Recv from `Recv(context.Context) (Address, proto.Message, error)` to `Recv(context.Context) (Request, error)` ?

Reply:
Also, if this is not likely that we add more things to `Request`, I would find it better to try keeping this interface simple and not use a new struct: `Process(req Request, addr Address) (resp proto.Message, err error)`, as you are already doing with Receiver.Recv

Reply:
As discussed, I think it will be useful later.